rs/nns/governance/api/src/pb.rs

```diff
@@ -1,8 +1,8 @@
 use crate::pb::v1::{
-    governance::migration::MigrationStatus, governance_error::ErrorType, manage_neuron_response,
-    neuron::DissolveState, CreateServiceNervousSystem, GovernanceError, ManageNeuronResponse,
-    NetworkEconomics, Neuron, NeuronState, NeuronsFundEconomics,
-    NeuronsFundMatchedFundingCurveCoefficients, VotingPowerEconomics, XdrConversionRate,
+    governance::migration::MigrationStatus, governance_error::ErrorType, neuron::DissolveState,
+    CreateServiceNervousSystem, GovernanceError, NetworkEconomics, Neuron, NeuronState,
+    NeuronsFundEconomics, NeuronsFundMatchedFundingCurveCoefficients, VotingPowerEconomics,
+    XdrConversionRate,
```

Comment:
Once again, no human can reasonably cope with this cluster of a diff. Here is the skinny: deleted manage_neuron_response, and ManageNeuronResponse.